### PR TITLE
fix: fix incorrect name for health filter in catalog name

### LINKED ISSUE #1992: Catalog health filter is not shown (even though its selected) when clicking on health in catalog summary

### PR DESCRIPTION
**Pull request by mainawycliffe**
Fixes #1992

### CODE REVIEW

**Comment by moshloop** [comment]
@mainawycliffe this does not appear to work

**Comment by mainawycliffe** [comment]
@moshloop did you mean this or something else -> 
image

**Comment by moshloop** [comment]
@mainawycliffe - I get a `getConfigStatusColor is not defined` error

**Comment by mainawycliffe** [comment]
@moshloop let me take a look

File: src/components/Configs/ConfigSummary/Cells/ConfigSummaryHealthCells.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    return `/catalogs?configType=${type}`;
+  }, [groupBy, type]);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const statusLines = useMemo(() => {
+    const data: StatusInfo[] = Object.entries(value ?? {}).map(
+      ([key, value]) => {
+        return {
+          label: value,
+          url: `${urlBase}&health=${key}`,
```
Comment (moshloop) [inline src/components/Configs/ConfigSummary/Cells/ConfigSummaryHealthCells.tsx:34]:
```suggestion
          url: `${urlBase}&health=${key}:1`,
```

**Pull request merged**